image_explorer/image_explorer.py

```diff
@@ -279,24 +280,35 @@ def _make_url_absolute(self, url):
         lms_base = '{}://{}'.format(scheme, lms_base)
         return urljoin(lms_base, url)
 
-    def _inner_content(self, tag):
+    def _inner_content(self, tag, absolute_urls=False):
         """
         Helper met
         """
         if tag is not None:
-            return u''.join(html.tostring(e) for e in tag)
+            if absolute_urls:
+                return self._change_relative_url_to_absolute(u''.join(html.tostring(e) for e in tag))
+            else:
+                return u''.join(html.tostring(e) for e in tag)
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). `u''.join(html.tostring(e) for e in tag)` is written twice; we need to store it in a separate variable.